build-support/ci_lists/integration_chroot_blacklist.txt

```diff
@@ -1,8 +1,4 @@
-tests/python/pants_test/backend/jvm/subsystems:jar_dependency_management_integration
 tests/python/pants_test/backend/jvm/tasks/jvm_compile/java:cache_compile_integration
```

Comment:
I couldn't get this to easily work with chroot. Will have a dedicated PR for this. I suspect this could be why remoting the JVM tasks takes so long.